crates/consensus/auto-seal/Cargo.toml

```diff
@@ -16,6 +16,7 @@ workspace = true
 reth-chainspec.workspace = true
 reth-beacon-consensus.workspace = true
 reth-primitives.workspace = true
+reth-eth-wire-types.workspace = true
```

Comment:
I'd rather re-export this from the p2p crate because this is part of the public API there:

https://github.com/paradigmxyz/reth/blob/6e1521456623aa3dbec586e3aa48b642c1fe28d5/crates/net/p2p/src/headers/client.rs#L13-L20